test/e2e_tests/utils/userDataGenerator.ts

```diff
@@ -40,11 +40,13 @@ export const generateSecurePassword = (length: number = 8): string => {
 };
 
 export const generateLastName = (): string => {
-  return faker.person.lastName();
+  // Remove apostrophes from last names
```

Comment:
We had an issue with names like O'Hara. All the tests failed in that case since the selectors didn't work